modules/storages/spec/lib/api/v3/storages/storages_representer_rendering_spec.rb

```diff
@@ -68,6 +69,21 @@
         let(:title) { 'Connected' }
       end
     end
+
+    describe 'oauthApplication' do
+      it_behaves_like 'has a titled link' do
+        let(:link) { 'oauthApplication' }
+        let(:href) { "/api/v3/oauth_applications/#{oauth_application.id}" }
+        let(:title) { oauth_application.name }
+      end
+    end
+
+    describe 'oauthClientCredentials' do
+      it_behaves_like 'has an untitled link' do
+        let(:link) { 'oauthClientCredentials' }
+        let(:href) { nil }
+      end
+    end
```

Comment:
Some of the paths, leading to the link not being rendered are not covered. e.g having no client credentials defined. 

The effects of this change on `_embedded` are not speced.

Reply:
added